src/messages/bundleBuilt.js

```diff
@@ -8,8 +8,19 @@ import type { WebpackStats } from '../types';
 
 const chalk = require('chalk');
 const dedent = require('dedent');
+
 const path = require('path');
 
+const getBuildTime = webpackStats => {
+  const stats = webpackStats.toJson({ timing: true });
+  return stats.time
+    ? stats.time
+    : stats.children.reduce(
```

Comment:
What about to go for something like this: `Math.max(...stats.children.map(({time}) => time))`? I think it's more readable.